src/flightlog_parser.js

```
@@ -776,208 +915,671 @@ export function FlightLogParser(logData) {
-  for (const field of PARSE_INT_FIELDS) { HEADER_HANDLERS[field] = parseIntHandler; }
-  for (const field of CSV_FIELDS) { HEADER_HANDLERS[field] = csvHandler; }
-  for (const field of VERSION_CONDITIONAL_INT_FIELDS) { HEADER_HANDLERS[field] = versionCondIntHandler; }
-  for (const field of VERSION_CONDITIONAL_CSV_FIELDS) { HEADER_HANDLERS[field] = versionCondCsvHandler; }
-  for (const field of ACCEL_LIMIT_FIELDS) { HEADER_HANDLERS[field] = accelLimitHandler; }
-  for (const field of DIVIDE_100_FIELDS) { HEADER_HANDLERS[field] = divide100Handler; }
-  for (const field of STRING_FIELDS) { HEADER_HANDLERS[field] = stringHandler; }
+  }
+  const noopHandler = () => {}
+
+  const HEADER_HANDLERS = {}
+  for (const field of PARSE_INT_FIELDS) {
```

ESLint

[error] Missing semicolon.

(semi)